### DISCUSSION TITLE
Multiple preview domains per-environment

### DISCUSSION THREAD

**Discussion by timkelty** [craftcms/cloud#23] [Feature Requests & Enhancements]
If you want to test multi-site/multi-domain in non-production, you have to set domains up, because you only get one preview domain. Would be nice if you could either generate more preview domains, or maybe you get a preview domain for every domain you create?

**Comment by tvd0x2a** [comment]
+1. We have a multisite application with 4 domains for each environment. For example, this is what our dev environment looks like:

`dev-admin.example.com` -> cp
`dev-alaska.example.com` -> site 1
`dev-newyork.example.com` -> site 2
`dev-www.example.com` -> site 3

We'd love to have some way to ensure that our multisite configuration will work with the different domains in dev during a trial. We don't care about the format of the domains—simply having 4 per environment would work for us. Thanks!

**Comment by jamesmacwhite** [comment]
Also adding +1, on the basis of multi-site that use more than one TLD domain configured. At the moment you'd be limited to a base URL off the single preview domain, which works, but perhaps not as easy to test between dev, staging, production.
